python_modules/dagster/dagster/_core/definitions/asset_selection.py

```diff
@@ -389,6 +416,10 @@ def from_string(cls, string: str) -> "AssetSelection":
                 selection = key_selection
             return selection
 
+        elif string.startswith("tag:"):
```

Comment:
Assuming that this is called from CLI? Is there documentation for this syntax anywhere?